<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="sidebar_enaplic" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(186, 24, 27, 255), stop:1 rgba(102, 7, 8, 255));
}</string>
    </property>
    <widget class="QLabel" name="logo_enaplic">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../static/resource.qrc">:/image/image/LOGO.jpeg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="main_area" native="true">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>-1</y>
      <width>641</width>
      <height>601</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
    background-color: #ffffff;
	font-style: &quot;Segoe UI&quot;;
 	color: #0B090A;
}

QLabel {
    color: #0B090A;
    font-size: 11px;
    font-weight: bold;
}

QLineEdit {
    background-color: #DFE0E2;
    padding-left: 15px;
    border-radius: 18px;
	font-size: 14px;
}

QPushButton {
    background-color: #fbba72;
    border: 2px;
    border-radius: 18px;
    font-size: 14px;
    font-weight: regular;
}

QPushButton:hover {
    background-color: #e5383b;
    color: #EEEEEE
}

QPushButton:pressed {
    background-color: #6703c5;
    color: #EEEEEE;
}</string>
    </property>
    <widget class="QLineEdit" name="user_field">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>221</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>221</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_field">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>221</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>221</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>18</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>330</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>221</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover {
    background-color: #399918;
    color: #EEEEEE;
	font-style: &quot;Segoe UI&quot;;
}</string>
     </property>
     <property name="text">
      <string> Entrar</string>
     </property>
     <property name="icon">
      <iconset resource="../static/resource.qrc">
       <normaloff>:/icon/icon/entrar (2).png</normaloff>:/icon/icon/entrar (2).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_register">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>390</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>221</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover {
    background-color: #00b4d8;
    color: #EEEEEE
}</string>
     </property>
     <property name="text">
      <string> Criar uma conta</string>
     </property>
     <property name="icon">
      <iconset resource="../static/resource.qrc">
       <normaloff>:/icon/icon/add-user-64.ico</normaloff>:/icon/icon/add-user-64.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_forget_password">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>450</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>221</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover {
    background-color: #ff6700;
    color: #EEEEEE
}</string>
     </property>
     <property name="text">
      <string> Esqueceu sua senha?</string>
     </property>
     <property name="icon">
      <iconset resource="../static/resource.qrc">
       <normaloff>:/icon/icon/cadeado (1).png</normaloff>:/icon/icon/cadeado (1).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="user_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Usu√°rio</string>
     </property>
    </widget>
    <widget class="QLabel" name="password_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="QLabel" name="logo_enaplic_50">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>510</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../static/resource.qrc">:/image/image/logo_enaplic_50_anos.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>0</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../static/resource.qrc">
       <normaloff>:/icon/icon/excluir.png</normaloff>:/icon/icon/excluir.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="window_title_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: transparent;
    color: #a4161a;
    font-size: 28px;
    font-weight: bold;
	font-style: &quot;Segoe UI&quot;;
}</string>
     </property>
     <property name="text">
      <string>Seja bem-vindo(a) ao Eureka!</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../static/resource.qrc"/>
 </resources>
 <connections/>
</ui>
